Readme.md

```diff
@@ -65,8 +65,8 @@ Setting read only properties results in a thrown `DBusException`.
 You can register a delegate into a `MessageRouter` and a main loop in order to handle messages.
 After that you can request a name so that other DBus clients can connect to your program.
 
-You can return a `Tuple!(args)` to return multiple values (multiple out values in XML) or
-return a `Variant!DBusAny` to support returning any dynamic value.
```

Comment:
If you wrap this you should also wrap the next change

Reply:
Thinking it over once again... I made a mistake here. Returning a struct is possible, but in this particular case, it will not elicit the same behavior as returning a `Tuple`. I will revert this change.